modules/ibi-image-based-install-cluster-guide.adoc

```diff
@@ -15,6 +15,8 @@ For a successful image-based installation and deployment, see the following guid
 
 * If your cluster deployment at the edge of the network requires a proxy configuration, you must create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
```

Comment:
This is also out of scope but perhaps more worthwhile to address in this PR if you can, but the two `==` headings above this line do not have any IDs on them, which they need to have (and with a `_{context}` at the end)